book/src/getting-started/3-exposing-to-qml.md

```diff
@@ -26,16 +26,22 @@ There are two notable changes compared to a normal Qt application though:
 {{#include ../../../examples/qml_minimal/cpp/main.cpp:book_qml_register}}
```

Comment:
there is another Qt 5 doc higher up here to the "Qt documentation"

Reply:
Updated to Qt6